### DISCUSSION TITLE
Multiple Monitor Layouts

### DISCUSSION THREAD

**Discussion by Farmerbrown87** [gerritdevriese/kzones#35] [Ideas]
Hi, 

Would it be possible to have layouts assigned per monitor? I have three 1920 x 1080 monitors with one of the oriented vertically, effectively  

1. 1920 x 1080 
2. 1920 x 1080
3. _1080 x 1920_  

I would like to assign a similar but vertically oriented layout to that third monitor without switching layouts via the keyboard shortcut.

Many thanks

**Comment by robjhyndman** [comment]
Even with two monitors of the same size, it is sometimes useful to have different layouts. e.g., I like to use the centre 2/3 of each monitor for one window, and the outer 1/3 of each monitor for another.

**Reply by gerritdevriese** [reply]
Have you looked at [this](https://github.com/gerritdevriese/kzones?tab=readme-ov-file#track-active-layout-per-screen) option? Doesn't this solve your issue?
The only drawback is that you'll need to set the correct layouts at boot manually.

**Reply by robjhyndman** [reply]
Oh, that option wasn't showing up for me. Now updated to the latest version and I can see it. Thanks
